lib/core/include/irods/irods_logger.hpp

```diff
@@ -179,22 +182,19 @@ namespace irods::experimental
 
                 impl() = default;
 
-                impl(const impl&) = delete;
-                impl& operator=(const impl&) = delete;
-
-                bool should_log() const noexcept
+                [[nodiscard]] auto should_log() const noexcept -> bool
                 {
                     return Level >= logger_config<Category>::level;
                 } // should_log
 
-                std::string utc_timestamp() const
+                [[nodiscard]] auto utc_timestamp() const -> std::string
                 {
                     // clang-format off
                     using clock_type      = std::chrono::system_clock;
                     using time_point_type = std::chrono::time_point<clock_type>;
                     // clang-format on
 
-                    timespec ts;
+                    timespec ts; // NOLINT
```

Comment:
Should we hold off on using `NOLINT` until we've decided on our clang-tidy configuration? Just feeling out the room here ;)

Reply:
I'll remove these so people can see what clang-tidy was complaining about. And yes, the use of NOLINT should be avoided until we settle on a clang-tidy config.

Reply:
Done.